form_validation.js

```diff
@@ -0,0 +1,14 @@
+const submitBtn = document.querySelector('.seeProject');
+const validateEmail = (email) => {
+  if (email.match(/^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/)) {
+    return true;
+  }
+  return false;
+};
+
+submitBtn.addEventListener('click', () => {
+  const emailVal = document.getElementById('mail').value;
+  if (!validateEmail(emailVal)) {
```

Comment:
- [ ] Hello and nice work on adding the form however the validation process is not quite the one you have implemented, I suggest visiting the requirement page again and reading it.

![Capture web_6-9-2022_12576_github com](https://user-images.githubusercontent.com/61562626/188628958-dde2b086-5c6b-47d5-93d2-7885b1e4bc5e.jpeg)

Reply:
solved